packages/ipfs/src/core/preload.js

```diff
@@ -4,29 +4,58 @@ const toUri = require('multiaddr-to-uri')
 const debug = require('debug')
 const CID = require('cids')
 const shuffle = require('array-shuffle')
-const AbortController = require('abort-controller')
+const { AbortController } = require('abort-controller')
```

Comment:
This was necessary because:

1. TS does not treat ES default export as `module.exports` in CommonJS. Which on it's own would not have being issues, but for a fact that abort-controller module provides `.d.ts` file that provides type signature that tells TS it is an ES module. In the end just using destracturing here addresses everything as module happens to export same value as `AbortController` as well.